Dockerfile

```diff
@@ -18,7 +18,7 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-FROM quay.io/bluecat/gateway:24.1.0
```

Comment:
Because we use the official GW image so I think we should use the latest version instead of a particular version
quay.io/bluecat/gateway:latest

Reply:
just a minor comment, feel free to resolve it.